gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -96,7 +96,7 @@ class AkkaGrpcPlugin implements Plugin<Project>, DependencyResolutionListener {
                                 option "server_power_apis=${extension.serverPowerApis}"
                                 option "use_play_actions=${extension.usePlayActions}"
                                 option "extra_generators=${extension.extraGenerators.join(';')}"
-                                option "logfile=${project.projectDir.toPath().relativize(logFile).toString()}"
+                                option "logfile=${logFile.toAbsolutePath().toString()}"
```

Comment:
```suggestion
                                option "logfile=${project.rootProject.projectDir.toPath().relativize(logFile).toString()}"
```
Maybe inserting rootProject will resolve the problem? rootProject will return the current project when it is a single project, and will return the root project when this is a sub project.
I have not tested it yet, but looking at the code and options available on project, this would be my first guess

Reply:
ok never mind that does not do what I hoped it would do

Reply:
Ok that did not do what I expected it would do, now it works when the task is run from the main project, but no longer works when it is run from the subproject.
But it is something, it generates the correct path, now only need to find out which should be used when...